engine/src/main/java/com/redhat/cloud/notifications/routers/EndpointTestResource.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.events.FromCamelHistoryFiller;
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.models.event.TestEventHelper;
+import com.redhat.cloud.notifications.routers.endpoints.EndpointTestRequest;
+import io.vertx.core.json.Json;
```

Comment:
This class is part of the test package of the common module, so it is not accessible by the source classes. Do you think I should move it to the sources so that we can use it in the main classes as well?

Reply:
My bad, you can disregard my comment.